fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java

```diff
@@ -647,4 +664,188 @@ private String getAllHeaders(HttpServletRequest request) {
         }
         return headers.toString();
     }
+
+    /**
+     * Request body example:
+     * {
+     *     "label": "test",
+     *     "tableToPartition": {
+     *         "tbl_test_spark_load": ["p1","p2"]
+     *     },
+     *     "properties": {
+     *         "strict_mode": "true",
+     *         "timeout": 3600000
+     *     }
+     * }
+     *
+     */
+    @RequestMapping(path = "/api/ingestion_load/{" + CATALOG_KEY + "}/{" + DB_KEY
+            + "}/_create", method = RequestMethod.POST)
+    public Object createIngestionLoad(HttpServletRequest request, HttpServletResponse response,
```

Comment:
Only Master FE can handle all these requests.
So need to add forward logic on non-master FE.

Reply:
done, same logic has been added to update action